tee-worker/client-api/parachain-api/prepare-build/interfaces/trustedCalls/definitions.ts

```diff
@@ -0,0 +1,103 @@
+export default {
+    types: {
+        WorkerRpcReturnValue: {
+            value: "Vec<u8>",
+            do_watch: "bool",
+            status: "DirectRequestStatus",
+        },
+        TrustedOperation: {
+            _enum: {
+                indirect_call: "(TrustedCallSigned)",
+                direct_call: "(TrustedCallSigned)",
+                get: "(Getter)",
+            },
+        },
+        TrustedCallSigned: {
+            call: "TrustedCall",
+            index: "u32",
+            signature: "LitentryMultiSignature",
+        },
+        Getter: {
+            _enum: {
+                public: "(PublicGetter)",
+                trusted: "(TrustedGetterSigned)",
+            },
+        },
+        PublicGetter: {
```

Comment:
Getters are not calls, maybe it would be worth to split it too.

Reply:
Thank you. Please help me correct any remaining misclassifications if there are any.